MUI_Core/Modules/Chat/Chat.lua

```diff
@@ -405,14 +407,7 @@ function C_ChatModule:OnEnable(data)
   local changeGameFont = db.global.core.changeGameFont;
```

Comment:
These are the 2 lines you are missing and should be in `SetUpAllBlizzardFrames` or maybe in `SetUpBlizzardChatFrame` - your choice.